templates/admin/workers/index.html.ep

```diff
@@ -75,6 +76,11 @@
                         <td class="version">
                             <%= exists $worker->{properties} && $worker->{properties}->{ISOTOVIDEO_INTERFACE_VERSION} ? $worker->{properties}->{ISOTOVIDEO_INTERFACE_VERSION} : 'unknown' %>
                         </td>
+                        <td class="action">
+                            % if ( is_admin && ($worker->{status} eq 'dead' && !$worker->{job} ) ) {
```

Comment:
Please format the template code like tidy would do. So no spaces after `(` and before `)`.

And by the way, run `script/tidy` to format your changes in general.